Src/PerformanceCollector/Filtering/AccumulatedValue.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Filtering
+{
+    using System.Collections.Concurrent;
+
+    /// <summary>
+    /// Accumulator for operationalized metrics.
+    /// </summary>
+    internal class AccumulatedValue
+    {
+        public readonly ConcurrentStack<double> Value = new ConcurrentStack<double>();
```

Comment:
Done. Renamed AccumulatedValue to AccumulatedValues.
ConcurrentStack is a highly performant spin-lock collection. The ordering of elements doesn't matter, so we don't actually need the stacking logic. There are a bunch of such collections, each varying in internal implementation quite significantly. This particular implementation is the one I've researched at one point, and it has proven itself well in production code, so I chose it here.